web/gen/web_columns.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+import json
+import re
+from pathlib import Path
+
+
+def extract_flow_columns(filename):
+    try:
+        with open(filename, "r") as file:
+            content = file.read()
+    except FileNotFoundError:
+        print(f"Error: File '{filename}' not found.")
+        return []
```

Comment:
This here is actively harmful error handling. First this case should never happen; and second, if it happens, it should not silently swallow the error.